### PR TITLE
Fix registration bug

### PR DESCRIPTION
**Pull request by Johnny850807**
Fix registration bug

### CODE REVIEW

File: spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java [hunk 1]
```diff
@@ -38,9 +40,11 @@ public void run(ApplicationArguments args) {
     }
 
     private Void registerService(RetryContext retryContext) throws RestClientException {
+        log.info("Dixit is registering itself to the Lobby. (Retry={})", retryContext.getRetryCount());
         Registration registration = new Registration(name, serviceHost, MIN_NUMBER_OF_PLAYERS, MAX_NUMBER_OF_PLAYERS);
         registration.addOption(new Registration.Option("winningScore", "range", 25, 35, 5));
         lobbyServiceDriver.registerService(registration);
+        log.info("Dixit registered itself to the Lobby successfully.");
```
Comment (wallychen0705) [inline spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java:47]: NIT: Can we use an aspect to log some information for debugging?

File: spring-boot-dixit/src/main/java/tw/wally/dixit/Registration.java [hunk 2]
```diff
@@ -9,6 +10,7 @@
 /**
  * @author - wally55077@gmail.com
  */
+@Data
```
Comment (wallychen0705) [inline spring-boot-dixit/src/main/java/tw/wally/dixit/Registration.java:13]: `@Getter`

**Review by wallychen0705 — CHANGES_REQUESTED** [review]
Review done.

File: spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java (hunk 1 above)
Comment (Johnny850807) [reply spring-boot-dixit/src/main/java/tw/wally/dixit/DixitApplicationRunner.java:47]: No need for that, this is just a runner....

**Pull request merged**